packages/dds/sequence/src/revertibles.ts

```diff
@@ -158,17 +158,25 @@ export function appendChangeIntervalToRevertibles(
 	revertibles: SharedStringRevertible[],
 ) {
 	const startSeg = previousInterval.start.getSegment() as SharedStringSegment;
+	const startType =
+		startSeg.removedSeq !== undefined
+			? ReferenceType.SlideOnRemove
+			: ReferenceType.StayOnRemove | ReferenceType.RangeBegin;
```

Comment:
I don't think we actually do anything with RangeBegin and RangeEnd on these revertible localRefs. Not really related to this change.